README.adoc

```diff
@@ -11,12 +11,13 @@ projects: []
 :toc:
 :icons: font
 :source-highlighter: prettify
-:project_id: draft-gs-template
+:project_id: gs-spring-cloud-gcp
 This guide walks you through the process of creating a Spring application.
 
 == What you'll build
 
-You'll build a Spring application.
+A Spring Boot application that exchanges messages through Google Cloud Pub/Sub. It uses the Spring
+Integration GCP Pubsub channel adapters to integrate with Google Cloud Pub/Sub.
```

Comment:
Here might be better to replace, but the other mentions to Pubsub are intentional, since the name of the library (i.e., the Maven artifact) actually reads Pubsub and not Pub/Sub.
Should I still go ahead with the global renaming?